app/main.py

```diff
@@ -1,2 +1,109 @@
+from typing import Any
+from dataclasses import dataclass
+from collections import defaultdict
+
+INITIAL_CAPACITY = 8
+LOAD_FACTOR = 2 / 3
+
+
+@dataclass
+class Node:
```

Comment:
Node can be defined inside Dictionary since it is part of its implementation

Reply:
resolved